tools/test/topology/test-all.m4

```diff
@@ -60,18 +60,18 @@ PIPELINE_PCM_ADD(sof/pipe-TEST_PIPE_NAME-capture.m4,
 # SSP port TEST_DAI_PORT is our only pipeline DAI
 #
 
-# playback DAI is SSP TEST_DAI_PORT using 2 periods
+# playback DAI is SSP TEST_DAI_PORT using 3 periods
```

Comment:
This sets periods for all tests ? Should we not just target SRC tests ?

Reply:
These test topologies need a complete rewrite, we have no more need for the I2S formats collection, but I'll see if I can just change SRC. ASRC should be good with 2 periods.

Reply:
There's now check for test pipe name and change periods 3 if src.

Reply:
ok, we should move these test topologies to topology 2 @ranj063 @juimonen  fyi.